python/lancedb/remote/client.py

```diff
@@ -35,29 +35,32 @@ def wrapped(self, *args, **kwargs):
 
 @attr.define(slots=False)
 class RestfulLanceDBClient:
-    url: str
+    db_name: str
+    region: str
+    api_key: str
```

Comment:
we should make sure this field doesn't get accidentally printed somewhere.

something like 
```
class Password(str):
    def __repr__(self):
        return "****"

   def __str__(self):
        return "****"
```

Reply:
can i just do __repr__ and __str__ on this Query object?